### PR TITLE
Replace atom with vs code

### LINKED ISSUE #1: replace Atom
Atom is sunset 2022-12-15 https://github.blog/2022-06-08-sunsetting-atom/

We need to replace it with something else. Very sad.

EDIT: We will use vscode — available on Windows, macOS, Linux.

### PR DESCRIPTION
**Pull request by VOD555**
Fix #1 
We decide to switch from atom to vs code for the class. We need to update the files including text `editor/atom`.

- [x] [week-01.md](https://github.com/Py4Phy/PHY432/blob/main/_modules/week-01.md)
- [x] [environment_installation.md](https://github.com/Py4Phy/PHY432/blob/main/modules/setup_unix/environment_installation.md)
- [x] [Unix_Shell.md](https://github.com/Py4Phy/PHY432/blob/main/modules/setup_unix/Unix_Shell.md)
- [x] [Intermediate_Unix_Shell.md](https://github.com/Py4Phy/PHY432/blob/main/modules/setup_unix/Intermediate_Unix_Shell.md)
- [x] [Git_basics.md](https://github.com/Py4Phy/PHY432/blob/main/modules/git/Git_basics.md)

### CODE REVIEW

File: modules/git/Git_basics.md
Comment (orbeckst) [inline modules/git/Git_basics.md]: Need to initially install the `code` command, see https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line

Comment (orbeckst) [reply modules/git/Git_basics.md]: Does `code` work when installed on Linux?

File: modules/git/Git_basics.md
Comment (orbeckst) [inline modules/git/Git_basics.md]: If we refer to the command itself then it would be `code` (no "vs"). If we refer to the name then "vs code" or to be clearer use the official name "Visual Studio Code".

File: modules/git/Git_basics.md
Comment (orbeckst) [inline modules/git/Git_basics.md]: use `code`

File: modules/git/Git_basics.md
Comment (orbeckst) [inline modules/git/Git_basics.md]: `code`

File: modules/python/HelloWorld.md
Comment (orbeckst) [inline modules/python/HelloWorld.md]: Visual Studio Code

File: modules/setup_unix/Intermediate_Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Intermediate_Unix_Shell.md]: `code`

File: modules/setup_unix/Intermediate_Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Intermediate_Unix_Shell.md]: Visual Studio Code (or your favorite editor)

File: modules/setup_unix/Intermediate_Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Intermediate_Unix_Shell.md]: `code`

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code (command `code`)

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code (command `code`)

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/Intermediate_Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Intermediate_Unix_Shell.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code (`code`)

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: VS Code --> Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]: Visual Studio Code

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md:414]: On macOS, additional commands need to be installed, see https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line.

On Linux and Windows, the `code` command should be available if you followed the standard installation instructions.

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thank you for the big update. Could you please do the following (see inline comments):
- use Visual Studio Code instead of vs code when we refer to the name of the editor and the app
- use \`code\` (i.e., `code`) when we refer to the command that is invoked from the command line
- add a note in _From the shell_ on how to enable `code` in macOS.
-

File: modules/git/Git_basics.md
Comment (VOD555) [reply modules/git/Git_basics.md]: It works for me.

Comment (orbeckst) [reply modules/git/Git_basics.md]: great!

Comment (orbeckst) [reply modules/git/Git_basics.md]: Then we only need the special instructions for macOS (which I tried).

Comment (VOD555) [reply modules/git/Git_basics.md]: I'll put these instructions in [environment_installation.md](https://github.com/Py4Phy/PHY432/blob/main/modules/setup_unix/environment_installation.md)

**Comment by orbeckst** [comment]
@VOD555 please ping me when you need another review — I have a million other things to do and will just look when you tell me. Thanks!

**Comment by VOD555** [comment]
@orbeckst It should be ok for another review.

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]:
```suggestion
We need to initially install the `code` command so that we can launch Visual Studio Code from the
```

File: modules/setup_unix/environment_installation.md
Comment (orbeckst) [inline modules/setup_unix/environment_installation.md]:
```suggestion
3. a [text editor](#text-editor) (by default, *Visual Studio Code* / `code`)
```

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]:
```suggestion
6. Open `notes/TODO.txt` in _Visual Studio Code_ and add a note to item 1 too look in the
```
Open `notes/TODO.txt` in *Visual Studio Code* and add a note to item 1 too look in the

File: modules/setup_unix/Unix_Shell.md
Comment (orbeckst) [inline modules/setup_unix/Unix_Shell.md]:
```suggestion
Run *Visual Studio Code* by either launching it from a menu or from the commandline [^4]
```

File: modules/python/HelloWorld.md
Comment (orbeckst) [inline modules/python/HelloWorld.md]:
```suggestion
Using you favorite editor (for instance, *Visual Studio Code*, i.e., the `code` command), create a text file named `hello.py` in the
```

File: modules/git/Git_basics.md
Comment (orbeckst) [inline modules/git/Git_basics.md]:
```suggestion
     for details, especially for *Visual Studio Code*.
```

**Review by orbeckst — APPROVED** [review]
Thanks for addressing my comments, lgtm (after minor changes).

**Pull request merged**
